index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <link rel="stylesheet" href="/dist/tutor.css" />
+    <script src="http://cdn.mathjax.org/mathjax/2.3-latest/MathJax.js?config=MML_HTMLorMML"></script>
```

Comment:
Yep, that's the major reason why I tried to avoid MathJax. It has its own loader (it made a loader before loaders/combiners existed) which cannot be combined into one file plus it loads images/fonts/etc depending on the browser. It's so difficult to include that cnx (which also minifies into 1 file) tried to host a local copy of MathJax (instead of the CDN) and gave up.